README.md

```diff
@@ -90,8 +90,8 @@ will be created in the "build" directory created automatically by
 Meson. If you want statically linked libraries, you can add
 `-Dstatic-linkage=true` option to the Meson command.
 
-If you also want to build zimwriterfs tool (which requires some extra
-dependencies) add `-Dwith-zimwriterfs=true` option.
+If you want to disable building zimwriterfs (to not install unneeded
+dependencies if you not need it) add `-Dwith-zimwriterfs=false` option.
```

Comment:
Please squash this commit directly with the commit adding the option.

Reply:
Done, commits are revisited:

0b13217e567f  - droped ( update README - will not change, no new option added)
cba95763f08d0 -  edited (moving build dependencies to root, turning off on Windows)
4692603b6f46ebd4  - droped (making writer optional)
1ad717098898  - kept (Moving tools.(cpp|h) to src)
fed07899802ca12  - droped (turning off building on Windows)
4fe0d367eb88  - kept (Integrate sources into tree)